client/src/pages/Availability/Availability.tsx

```diff
@@ -0,0 +1,297 @@
+          sx={{ marginRight: '1rem', fontWeight: 700 }}
+          name="schedule"
+          value={selectedSchedule}
+          onChange={(e) => handleChange(e)}
+        >
+          {schedules.map((schedule) => generateSchedules(schedule.name, schedule.id))}
+        </Select>
+      </FormControl>
+      <Button
+        variant="contained"
+        color="primary"
+        sx={{ height: '4em' }}
+        disableElevation
+        onClick={() => {
+          setNewSchedule(true);
+          setDropDownSelected(false);
+        }}
+      >
+        + New Schedule
+      </Button>
+      {newSchedule && (
+        <Formik
+          initialValues={{
+            name: '',
+            days: {
+              monday: {
+                active: false,
+                startTime: '10:00',
+                endTime: '22:00',
+              },
+              tuesday: {
+                active: false,
+                startTime: '10:00',
+                endTime: '22:00',
+              },
+              wednesday: {
+                active: false,
+                startTime: '10:00',
+                endTime: '22:00',
+              },
+              thursday: {
+                active: false,
+                startTime: '10:00',
+                endTime: '22:00',
+              },
+              friday: {
+                active: false,
+                startTime: '10:00',
+                endTime: '22:00',
+              },
+              saturday: {
+                active: false,
+                startTime: '10:00',
+                endTime: '22:00',
+              },
+              sunday: {
+                active: false,
+                startTime: '10:00',
+                endTime: '22:00',
+              },
+            },
+          }}
+          onSubmit={handleSubmit}
+        >
+          {({ values, setFieldValue, handleChange, handleSubmit, isSubmitting }) => (
+            <form onSubmit={handleSubmit} style={{ marginTop: '20px' }}>
+              <FormInput
+                id="scheduleSelect"
+                label="New Schedule"
+                margin="dense"
+                name="name"
+                sx={{ width: '20%' }}
+                placeholder="Schedule name"
+                autoComplete="name"
+                autoFocus
+                value={values.name}
+                onChange={handleChange}
+              />
+              <Typography variant="h3" sx={{ fontWeight: 500, fontSize: '2.4em', marginTop: 4, marginBottom: 2 }}>
+                Set your weekly hours
+              </Typography>
+              <Box sx={{ border: '1px solid #dbdbdb', borderWidth: '1px 1px 0px 1px' }}>
+                {week.map((day) => CreateAvailInterface(day, values, setFieldValue))}
+              </Box>
+              <Box textAlign="center" marginTop={5}>
+                <Button
+                  type="submit"
+                  onClick={() => {
+                    setCheck(1);
+                    window.location.reload();
```

Comment:
Can we take another approach here rather than reloading the entire window?

Reply:
Done the changes